spec/features/search_by_name.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe 'Food Search' do
+  describe 'Serach' do
+    it 'allows user to search by name' do
+      visit root_path
+
+      fill_in :search, with: 'sweet potatoes'
+      click_button 'Search'
+
+      expect(page.status_code).to eq 200
```

Comment:
Good test so far!

Reply:
To flesh this out, your `expect(page).to have_content...` lines should include the expected objects that we might see back from the API.